billy_metadata/hi.py

```diff
@@ -0,0 +1,119 @@
+# from billy.utils.fulltext import pdfdata_to_text, text_after_line_numbers
+# from .bills import HIBillScraper
+# from .legislators import HILegislatorScraper
+# from .events import HIEventScraper
+
+settings = dict(SCRAPELIB_TIMEOUT=300)
```

Comment:
I don't think this line does anything here, but it should probably be restored in `openstates/hi/__init__.py`.